opentracing-jdbc/opentracing-jdbc/src/test/java/org/zalando/opentracing/jdbc/span/CachingPeerParserTest.java

```diff
@@ -0,0 +1,37 @@
+package org.zalando.opentracing.jdbc.span;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+class CachingPeerParserTest {
+
+    private final PeerParser parser = mock(PeerParser.class);
+    private final PeerParser unit = new CachingPeerParser(parser);
+
+    @BeforeEach
+    void setUp() {
+        when(parser.parse(any())).thenReturn(Peer.EMPTY);
+    }
+
+    @Test
+    void shouldDelegate() {
+        final Peer peer = unit.parse("");
+        assertEquals(Peer.EMPTY, peer);
+    }
+
+    @Test
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/whiskeysierra/tracer/pullRequest?prid=4563925)